src/gui/systray.cpp

```diff
@@ -223,6 +223,29 @@ bool Systray::useNormalWindow() const
     return cfg.showMainDialogAsNormalWindow();
 }
 
+bool Systray::darkMode() const
+{
+#if defined(Q_OS_MACOS)
+    return osXInDarkMode();
+// Windows: Check registry for dark mode
+#elif defined(Q_OS_WIN)
+    const auto darkModeSubkey = QStringLiteral("Software\\Microsoft\\Windows\\CurrentVersion\\Themes\\Personalize");
+    if (!Utility::registryKeyExists(HKEY_CURRENT_USER, darkModeSubkey)) {
+        return false;
+    }
+    const auto darkMode = Utility::registryGetKeyValue(HKEY_CURRENT_USER, darkModeSubkey, "AppsUseLightTheme");
+    switch (darkMode.toInt()) {
+    case 0:
+        return true;
+    case 1:
```

Comment:
Can there be other modes (2, 3, etc.)? If not, maybe remove a switch case and just cast `toBool` instead of `toInt()`?

Reply:
Good idea, changed it to a bool